make --export-pass format compatible with browserpass

edwintorok opened the pull request:
https://github.com/dannyvankooten/browserpass expects a multiline format with password first, and then a line with either login:, user: or username:.
Exporting passwords this way makes login form filling work if you have browserpass installed. Without it browserpass can't fill in the loginname, unless multiple accounts exist and the filename contains the account name.

unode commented:
Hi @edwintorok thanks for the pull request.

I wouldn't mind supporting `browserpass` but I don't want to break compatibility with the existing approach. There's other software out there that expects the password to be on the second line without any extra text around it.

The current patch as-is breaks that. I'd be ok with accepting a `--export-pass-compat browserpass` argument that sets this variant of output while keeping the default approach (`--export-pass-compat default`). Also please mention this in the Changelog and feel free to include yourself in the contributor list as part of this pull request.

edwintorok commented:
Updated my PR to be backwards compatible like you suggested and updated the Changelog.
I don't think its needed to update `AUTHORS` for a change small as this, anyone who wants to see all the authors can find it from `git shortlog`.

unode commented:
Thanks!
